doc/man7/property.pod

```diff
@@ -117,8 +117,8 @@ Ordering of optional clauses is not significant.
 =head2 Shortcut
 
 In order to permit a more concise expression of boolean properties, there
-is one short cut: a property name alone (e.g. "default") is
-exactly equivalent to "default=yes" in both definitions and queries.
+is one short cut: a property name alone (e.g. "myproperty") is
```

Comment:
A thought: should _myproperty_ be changed to something that is legal.  E.g. _my.property_.  User defined properties need a dot in them.

This is completely optional.